dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/transport/WriteQueue.java

```diff
@@ -31,17 +31,31 @@
 public class WriteQueue {
 
     static final int DEQUE_CHUNK_SIZE = 128;
-    private final Channel channel;
+    private volatile Channel channel;
```

Comment:
This `channel` cloud be deleted cuz parent channel can be got from `cmd`'s stream channel